dsgrid/config/dimension_mapping_base.py

```diff
@@ -66,7 +156,7 @@ class DimensionMappingReferenceModel(DSGBaseModel):
     )
     required_for_validation: Optional[bool] = Field(
         title="version",
-        description="Set to False if a given base-to-base dimension mapping is NOT required for input dataset validation; default is True",
+        description="Set to False if a given base-to-base dimension mapping is NOT required for input dataset validation; default is True",  # outdated, FIXME
```

Comment:
Can we fix as part of this PR?

Reply:
What is the new replacement term for base-to-base dimension mapping? Dimension associations?